src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,29 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("method completeOrder was called. Params: userId = {}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
         return order;
     }
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
-        // NOTE: In production ready code this method should fetch data from DB
-        // For test purpose we simplify this method and return dummy data
+        logger.info("getAllProductsFromShoppingCart method was called. Params: userId = {}",
+                userId);
         Product iphone = new Product("iPhone X", BigDecimal.valueOf(1199));
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("successful fetched data from DB. Params: products = {}", products);
```

Comment:
```suggestion
        logger.info("successful fetched data from DB. Params: userId = {}", userId);
```
I do not think logging DB is a good idea. What if DB will contain some secret data?
I think it will be better to log just income params. This way we can know who have access to the DB and will find the traitor)

Reply:
I think it depends on the case, here we have an order, no secret data, I'll leave it at that